src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,43 @@
+@import '~box-ui-elements/es/styles/variables';
+@import './ColorPickerControl';
+@import '../../../Controls';
```

Comment:
We should avoid importing other component stylesheets. Can we move the common variable to `../styles`?